src/matter/cluster/OccupancySensingCluster.ts

```diff
@@ -0,0 +1,78 @@
+/**
+ * @license
+ * Copyright 2022 The node-matter Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Attribute, OptionalAttribute, OptionalWritableAttribute, Cluster, Command, TlvNoArguments, TlvNoResponse } from "./Cluster";
+import { BitFlag, MatterApplicationClusterSpecificationV1_0, TlvBitmap, TlvInt16, TlvUInt16, TlvEnum, TlvField, TlvNullable, TlvObject, TlvSchema, TlvUInt8 } from "@project-chip/matter.js";
+
+/** @see {@link MatterApplicationClusterSpecificationV1_0} § 2.7.5.1 */
+export const OccupancyBitmap = TlvBitmap(TlvUInt8, {
+    occupied: BitFlag(0),
+});
+
+/** @see {@link MatterApplicationClusterSpecificationV1_0} § 2.7.5.2 */
+export const enum OccupancySensorType {
+    PIR = 0,
+    Ultrasonic = 1,
+    PIRAndUltrasonic = 2,
+    PhysicalContact = 3,
+}
+
+/** @see {@link MatterApplicationClusterSpecificationV1_0} § 2.7.5.3 */
+export const OccupancySensorTypeBitmap = TlvBitmap(TlvUInt8, {
+    PIR: BitFlag(0),
+    ultrasonic: BitFlag(1),
+    physicalContact: BitFlag(2)
+});
+
+/**
+ * Attributes and commands for Occupancy Detection.
+ *
+ * @see {@link MatterApplicationClusterSpecificationV1_0} § 2.7
+ */
+export const OccupancySensingCluster = Cluster({
+    id: 0x0406,
+    name: "OccupancySensing",
+    revision: 3,
+
+    /** @see {@link MatterApplicationClusterSpecificationV1_0} § 2.7.5 */
+    attributes: {
+       /** Bitmap, with only Bit 0 used.  1 = occupied, 0 = unoccupied. */
+       occupancy: Attribute( 0x0000, OccupancyBitmap ),
```

Comment:
Please do one last "code style change" ... For normal brackets we do not use sdpaces after the opening bracket and before the closing one. We just use spaces when using curly brackets "{ ... }" inside them if you put then in a "One-liner".

I think somewhen we add auto formattling tools then all this is much easier.

Reply:
OK - I changed all instances of ( XXX ) to (XXX)